editors/sc-ide/core/sc_introspection.cpp

```diff
@@ -72,7 +72,7 @@ bool Introspection::parse(const QString & yamlString )
 
     clear();
 
-    //qDebug("parsing introspection...");
+    qDebug("parsing introspection...");
```

Comment:
please do not remove code by commenting out. there is absolutely no point in commenting out this, btw